### DISCUSSION TITLE
Next 12.1.0 is ~2s slower per request than Next 11.1.4

### DISCUSSION THREAD

**Discussion by fierysolid** [vercel/next.js#35230] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

    Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 21.2.0: Sun Nov 28 20:28:41 PST 2021; root:xnu-8019.61.5~1/RELEASE_ARM64_T6000
    Binaries:
      Node: 16.13.2
      npm: 8.3.1
      Yarn: 1.22.17
      pnpm: 6.11.0
    Relevant packages:
      next: 12.1.1-canary.9
      react: 17.0.2
      react-dom: 17.0.2

### What browser are you using? (if relevant)

Chrome 97.0.4692.99 (Official Build) (arm64)

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

When I upgrade next to 12.1.0, the latency of requests increases ~2s. You can can see from the graph below that the down periods are when I'm using 11.1.4 and the higher periods are when I'm using 12.1.0.

<img width="828" alt="Screen Shot 2022-03-10 at 4 37 47 PM" src="https://user-images.githubusercontent.com/7339664/157759039-2fd4e41b-05f8-45c4-bf1a-69298217b879.png">

### Expected Behavior

Next 12.1.0 should maintain the same request performance as Next 11.1.4

### To Reproduce

Run request latency on 12.1.0 vs 11.1.4 to see the results.

**Comment by Marcelixoo** [comment]
Hey, @fierysolid! I'm just checking whether you found a way through this issue as I'm experiencing the same performance downgrade after upgrading Next.js from 9.4 to 12.1 :(

**Reply by fierysolid** [reply]
Sadly I haven't. I downgraded to 11 and haven't revisited this.
